src/lib/mongoBackend/MongoCommonSubscription.cpp

```diff
@@ -162,11 +162,15 @@ void setNotificationInfo(const Subscription& sub, orion::BSONObjBuilder* b)
 {
   if (sub.notification.type == ngsiv2::HttpNotification)
   {
-    b->append(CSUB_REFERENCE, sub.notification.httpInfo.url);
-    b->append(CSUB_CUSTOM,    sub.notification.httpInfo.custom);
+    b->append(CSUB_REFERENCE,     sub.notification.httpInfo.url);
+    b->append(CSUB_CUSTOM,        sub.notification.httpInfo.custom);
+    // FIXME httpTimeout is a long value, should cast to long or int?
```

Comment:
append() is a polymorphic method that can adapt to the different kind of number types used as argument. Thus, I guess the code is find as it is now.